frontend/src/components/TransactionTable/EditTransaction.js

```diff
@@ -47,7 +47,7 @@ function EditTransaction({ transaction, formatDate }) {
   const handleEdit = (e) => {
     e.preventDefault();
     const transaction = {
-      _id: transaction._id,
+      _id: e.target.id.value,
```

Comment:
did you revert since it was broken?

Reply:
yes, `transaction._id` does not work because cannot access `transaction`